lib/blpapi-wrapper.ts

```diff
@@ -279,8 +286,11 @@ export class Session extends events.EventEmitter {
         this.requests[correlatorId] = callback;
 
         this.openService(uri).then((): void => {
-            var responseEventName = reqName2RespName(name);
-            var requestName = name + 'Request';
+            if (!(name in REQUEST_NAMES_MAP)) {
```

Comment:
@yupinghu is making similar changes here in #73. Maybe we can land that first and rebase against that?